packages/angular-17/projects/components/README.md

```diff
@@ -0,0 +1,76 @@
+[![Storybook](https://img.shields.io/badge/docs-storybook-ff69b4)](https://tds-storybook.tegel.scania.com/)
+![](https://img.shields.io/github/license/scania-digital-design-system/tegel)
+![Status: Beta](https://img.shields.io/badge/status-beta-red)
+![npm](https://img.shields.io/npm/v/%40scania%2Ftegel-angular-17)
+
+# @scania/tegel-angular-17
+This is a ongoing project with the purpose to simplify the integration of Tegel in Angular apps.
+
+This guide [here](https://stenciljs.com/docs/angular) has served as base for the configuration.
+
+Official website: https://tegel.scania.com/
+
+Storybook: https://tds-storybook.tegel.scania.com/
+
+The design system supports the design and development of digital solutions at Scania. The purpose is to secure a coherent, premium brand and user experience across all of Scania's digital touchpoints.
+
+## Installation
+
+
+1. Run `npm install @scania/tegel-angular`
```

Comment:
Should this be updated to reflect the new npm package, so the users don't accidentally don't download the package that is not compatible with Angular 17?

Reply:
Good finding!